src/library/basetrackcache.cpp

```diff
@@ -381,6 +381,8 @@ void BaseTrackCache::getTrackValueForColumn(TrackPointer pTrack,
         trackValue.setValue(pTrack->getRating());
     } else if (fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_KEY) == column) {
         trackValue.setValue(pTrack->getKeyText());
+        QVariant keyCodeValue(static_cast<int>(pTrack->getKey()));
+        trackValue = KeyUtils::keyFromColumns(std::move(trackValue), std::move(keyCodeValue));
```

Comment:
I do not think this use of std::move and rvalue reference in the called function do what you think it will do, and it adds noise to the code.  Almost certainly, given the function signatures, the compiler will do the right thing and avoid extra copies by itself.  The function should just take const& and it will work fine
```suggestion
        trackValue = KeyUtils::keyFromColumns(trackValue, keyCodeValue);
```

Reply:
Yeah, I got carried a way a bit here 😄